pkgs/applications/audio/pd-plugins/timbreid/default.nix

```diff
@@ -1,30 +1,45 @@
-{ stdenv, fetchurl, unzip, puredata }:
+{ stdenv, fetchurl, unzip, puredata, fftwSinglePrec }:
 
 stdenv.mkDerivation rec {
-  version = "0.6.0";
+  version = "0.7.0";
   name = "timbreid-${version}";
 
   src = fetchurl {
     url = "http://williambrent.conflations.com/pd/timbreID-${version}-src.zip";
-    sha256 = "02rnkb0vpjxrr60c3hryv7zhyjpci2mi9dk27kjxpj5zp26gjk0p";
+    sha256 = "14k2xk5zrzrw1zprdbwx45hrlc7ck8vq4drpd3l455i5r8yk4y6b";
   };
 
-  buildInputs = [ unzip puredata ];
+  buildInputs = [ unzip puredata fftwSinglePrec ];
+
+  makeFlags = [
+    "CFLAGS= -I${puredata}/include/pd -Wall -W -g “-fPIC” --enable-float"
```

Comment:
Do not need unnecessary flags. Add necessary flags with `NIX_CFLAGS_COMPILE`. It looks like `--enable-float` belongs in the configure flags, not here. What are those fancy quotes around `-fPIC` doing?